### ISSUE TITLE
API Guide?

### ISSUE THREAD

**Issue by X-Raym** [Gravit-io/gravit-designer-plugin-starter#3]
Hi,

Is there an API guide which contains all the functions / methods / objects definitions available ?

Thanks !

**Comment by dennererthal01** [comment]
Hi @X-Raym, unfortunately we still don't have a guide, but we are working on it. Currently, we want people to try the new architecture so they can give us their feedback on it and we also can measure where we have to focus most. For now, we will be working together with everyone that wants to build a plugin and help as much as we can. 

Thanks

**Comment by FelixINX** [comment]
Hi @X-Raym and @dennererthal01 !
I also look forward to having documentation. But I see that it progresses, you have created a subdomain and a repo!
Thanks

**Comment by dennererthal01** [comment]
Yes @X-Raym we will start soon, and we are also aiming allow the community to contribute on this.

**Comment by SkyzohKey** [comment]
Any news about the API ?

**Comment by dennererthal01** [comment]
@SkyzohKey we had to postpone this because of bigger priorities on the app, we expect to be able give this topic full attention again soon, but we are unable to set a defined date right now.
